packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -650,166 +654,22 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
                     errorType,
                 });
                 const snapshotOptionsWithoutBlobs: ISnapshotOptions = { ...snapshotOptions, blobs: 0, mds: undefined, timeout: undefined };
-                return this.fetchSnapshotCore(snapshotOptionsWithoutBlobs, tokenFetchOptions);
+                const { odspSnapshot } = await fetchLatestSnapshotCore(
+                    this.odspResolvedUrl,
+                    this.getStorageToken,
+                    tokenFetchOptions,
+                    snapshotOptionsWithoutBlobs,
+                    this.logger,
```

Comment:
Consider creating child logger where fetchLatestSnapshotCore is called and adding property do differentiate these calls, i.e. such that it's clear from telemetry if the call is coming due to pre-fetching or not.

Reply:
I had child logger created in prefetch api with prefetch property.